persistence/src/commonMain/kotlin/com/wire/kalium/persistence/backup/DatabaseImporter.kt

```diff
@@ -54,6 +55,29 @@ class DatabaseImporterImpl(private val sqlDriver: SqlDriver) : DatabaseImporter
         restoreTable("MessageRestrictedAssetContent")
     }
 
+    private fun restoreConversations() {
+        // For some strange reason, when restoring the backup db, all last_read_date fields from backup db are hardcoded to 0. Therefore,
```

Comment:
@gongracr that's not when restoring, that's just how it is when you never read a conversation. The comment should just say 


> Before restoring a conversations, we need to set the last_read_date of the backup conversations to the last_read_date of the current conversations if it is more recent that what the backup states